server/src/test/java/com/broadcom/lsp/cobol/service/CobolTextDocumentServiceTest.java

```diff
@@ -437,6 +441,58 @@ void testImmediateClosingOfDocumentDoNotCauseNPE() {
     verify(communications, timeout(2000)).cancelProgressNotification(UseCaseUtils.DOCUMENT_URI);
   }
 
+  /**
+   * This method tests that after a shutdown request, {@link TextDocumentService} always return
+   * SHUTDOWN_RESPONSE
+   *
+   * @throws ExecutionException
+   * @throws InterruptedException
+   */
+  @Test
+  void whenShutdownIsFired_ThenNewRequestReturnInvalidResponse()
+      throws ExecutionException, InterruptedException {
+    CobolTextDocumentService service =
+        (CobolTextDocumentService)
+            LangServerCtx.getInjector().getInstance(TextDocumentService.class);
+    LanguageServer server = LangServerCtx.getInjector().getInstance(LanguageServer.class);
```

Comment:
Please, don't use this type of DI in tests

Reply:
done as per the rest of the test class

Reply:
working on it, seems like I overlooked

Reply:
done